scratch/evaluate_gaming_traffic.cc

```diff
@@ -72,7 +72,7 @@ void
 CalculateThroughput (Ptr<OutputStreamWrapper> stream)
 {
   Time now = Simulator::Now ();    /* Return the simulator's virtual time. */
-  double cur = (packetSink->GetTotalRx () - lastTotalRx) * 8.0 / 1e3;     /* Convert Application RX Packets to kb. */
+  double cur = (packetSink->GetTotalRx () - lastTotalRx) * 8.0 / computeThroughputPeriodicity;     /* Convert Application RX Packets to kb. */
```

Comment:
`computeThroughputPeriodicity` is 100, before you were dividing by 1e3, what are you trying to achieve?
If you want kilobits you should divide by 1e3, if you want kbps you should divide by the window duration (in [s]) and then by 1e3.

Reply:
yeah, you right. I forgot to convert it to kbps